README.md

```diff
@@ -236,485 +255,1824 @@ const {
-    });
-  });
+For more information about Embedded Messaging, read the [Embedded Messaging Oveview](https://support.iterable.com/hc/articles/23060529977364).
 
-  /* optionally logout the user when you don't need to hit the Iterable API anymore */
-  logout();
-})();
-```
+## How do I make API requests with the SDK?
 
-Now let's put it altogether with an Iterable API method:
+Follow these steps:
 
-```ts
-import { initialize, track } from '@iterable/web-sdk';
+1. In Iterable, [create a JWT-enabled, web API key](https://support.iterable.com/hc/articles/360043464871). 
+   The SDK will use this key to authenticate with Iterable's API endpoints. This
+   will ensure the SDK has access to all the necessary.  Save the API key and
+   its associated JWT secret, since you'll need them both.
 
-(() => {
-  const { setUserID, logout } = initialize(
-    'YOUR_API_KEY_HERE',
-    ({ email, userID }) =>
-      yourAsyncJWTGeneratorMethod({ email, userID }).then(
-        ({ jwt_token }) => jwt_token
-      )
-  );
+2. Work with your Engineering team to create a web service the SDK can call
+   to fetch a valid JWT token for the signed-in user. To learn more about how
+   to do this, read [JWT-Enabled API Keys](https://support.iterable.com/hc/articles/360050801231).
 
-  yourAsyncLoginMethod().then((response) => {
-    /* this code assumes you have some backend endpoint that will return a user's ID */
-    setUserID(response.user_id).then(() => {
-      document.getElementById('my-button').addEventListener('click', () => {
-        /* 
-          no need to pass a user ID to this endpoint. 
-          _setUserID_ takes care of this for you
-        */
-        track({ eventName: 'button-clicked' });
-      });
-    });
-  });
-})();
-```
+3. Use the API key to initialize the SDK, as described in [`initialize`](#initialize) 
+   and [`initializeWithConfig`](#initializewithconfig). When you initialize the 
+   SDK, to pass in a method that can call the web service (created in step 2)
+   to fetch a valid JWT token for the signed-in user.
 
-## How does the SDK pass up my email / user ID?
+4. To identify the user to the SDK, call `setEmail` or `setUserID` (returned by 
+   [`initialize`](#initialize) or [`initializeWithConfig`](#initializewithconfig)). 
+   The SDK uses the user's `email` or `userId` to fetch a valid JWT token from 
+   your server. 
 
-This SDK relies on a library called [Axios](https://github.com/axios/axios). For
-all outgoing XHR requests, the SDK utilizes [Axios interceptors](https://github.com/axios/axios#interceptors) 
-to add your user information to the requests.
+5. After the SDK successfully sets the user's `email` or `userId`, you can make 
+   API requests to Iterable. For example, you can call [`track`](#track) to track 
+   events, or [`getInAppMessages`](#getinappmessages) to fetch in-app messages.
+    
+    
+## How does SDK add the user's `email` or `userId` to the requests it makes to Iterable?
```

Comment:
I feel like this piece should be much higher up in the README. Any major functionality explanations like this should take precedent over method/type definitions and their references, imo.

Reply:
OK, changed to the following, and moved higher up:

```
# Using the SDK

In general, to use the SDK, you'll need to follow these steps:

1. In Iterable, [create a JWT-enabled, web API key](https://support.iterable.com/hc/articles/360043464871). 
   The SDK can use this key to authenticate with Iterable's API endpoints. This
   will ensure the SDK has access to all the necessary. Save the API key and
   its associated JWT secret, since you'll need them both.

2. Work with your Engineering team to create a web service the SDK can call
   to fetch a valid JWT token for the signed-in user. To learn more about how
   to do this, read [JWT-Enabled API Keys](https://support.iterable.com/hc/articles/360050801231).

3. [Install](#installation) the SDK in your web app.

4. Use the API key to initialize the SDK, as described in [`initialize`](#initialize) 
   and [`initializeWithConfig`](#initializewithconfig). When you initialize the 
   SDK, pass in a method that can call the web service (created in step 2)
   to fetch a valid JWT token for the signed-in user.

5. To identify the user to the SDK, call `setEmail` or `setUserID` (returned by 
   [`initialize`](#initialize) or [`initializeWithConfig`](#initializewithconfig)). 
   The SDK uses the user's `email` or `userId` to fetch a valid JWT token from 
   your server. 

6. After the SDK successfully sets the user's `email` or `userId` and the SDK
   fetches a JWT token, you can make API requests to Iterable. For example, you
   can call [`track`](#track) to track events, or [`getInAppMessages`](#getinappmessages) 
   to fetch in-app messages, etc. Other methods are described [below](#functions).
```